erpnext/erpnext_integrations/connectors/shopify_connection.py

```diff
@@ -335,13 +335,13 @@ def get_url(shopify_settings):
 
 	if not last_order_id:
 		if shopify_settings.sync_based_on == 'Date':
-			url = get_shopify_url("admin/api/2020-10/orders.json?limit=250&created_at_min={0}&since_id=0".format(
+			url = get_shopify_url("admin/api/2021-04/orders.json?limit=250&created_at_min={0}&since_id=0".format(
```

Comment:
Might be useful to store this in a global constant in the Shopify module and use that everywhere. It'll be easier to update in the future.

Reply:
Planned to be changed in future update. 

Right now it will create require circular imports, so it will create more mess. (or create a new file)

Reply:
Ah, I see. No problem then.